docs/pages/technical-specs/expo-updates-0.md

```diff
@@ -81,6 +88,10 @@ content-type: *
 * `expo-server-defined-headers` is an [Expo SFV](expo-sfv.md) dictionary. It defines headers that a client library MUST store until overwritten by a newer dictionary, and they MUST be included in every subsequent [manifest request](#manifest-request).
 * `cache-control` MUST be set to an appropriately short period of time. A value of `cache-control: private, max-age=0` is recommended to ensure the newest manifest is returned. Setting longer cache ages could result in stale updates.
 * `content-type` MUST be determined by _proactive negotiation_ as defined in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-3.4.1). Since the client library is [required](#manifest-request) to send an `accept` header with each manifest request, this will always be either `application/expo+json`, `application/json`; otherwise the request would return a `406` error.
+* `expo-signature` SHOULD contain the signature of the manifest to be used during the validation step of [code signing](#code-signing) if the request for the manifest contained the `expo-accept-signature` header. This is an [Expo SFV](expo-sfv-0.md) dictionary which MAY contain any of the following key value pairs:
+    * `sig` MUST contain the signature of the manifest. The name of this field matches that of `expo-accept-signature`.
```

Comment:
expo-expects-signature

Reply:
Maybe spell out `signature` here since we are not abbreviating it else where?

Reply:
https://www.ietf.org/archive/id/draft-ietf-httpbis-message-signatures-06.html seems to abbreviate it but in that spec it's a user-specified identifier identifying the key to the dictionary. I guess it depends on whether we want to allow for multiple signatures to be sent (signatures to be sent for multiple keys) as is specified in https://www.ietf.org/archive/id/draft-ietf-httpbis-message-signatures-06.html#name-the-accept-signature-field.